src/components/plots/EventsOverTimePlot.tsx

```diff
@@ -83,7 +83,7 @@ const EventsOverTimePlot: React.FC<PlotProps> = ({
     };
 
     return (
-        <div className={plotStyles.plotContainer}>
+        <div className={plotStyles.plotContainer} style={{ marginBottom: 0 }}>
```

Comment:
the margin at the bottom of this container made the component go longer than it needs to (there is nothing below this component)